make-release.sh

```diff
@@ -136,6 +136,14 @@ if [[ $TRIGGER_RELEASE -eq 1 ]]; then
   git branch release -f 
   git push origin release -f
```

Comment:
if this is a workflow dispatch (rather than on push to release branch) ... do we need this step?

Reply:
also probably want to remove "to trigger CI build" here since we don't want to fire on centos CI anymore.